data.sql

```diff
@@ -1,16 +1,29 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-2-3', 0, true, 10.23);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, true, 8);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-1-7', 1, false, 15.04);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-5-12', 5, true, 11);
+INSERT INTO owners (full_name, age)
+VALUES 
+  ('Sam Smith', 34),
+	('Jennifer Orwell', 19),
+	('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38)
+
 
+INSERT INTO species (name)
+VALUES 
+  ('Pokemon'),
+	('Digimon')
 
```

Comment:
- [ ] You need to modify tha animals table `species_id`. What I think you should do is to update the animals table and set the `species_id` to a specific value for all animals whose name ends with `mon` to `Digimon` and all others to `pokemon`

See Example:
```
UPDATE animals
SET species_id = (SELECT id from species WHERE name = 'Digimon')
WHERE name like '%mon';

UPDATE animals
SET species_id = (SELECT id from species WHERE name = 'Pokemon')
WHERE species_id IS NULL;
```

Reply:
Done